db/migrate/20220721233953_create_accounts.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccounts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :accounts do |t|
+      t.string :name, null: false
+      t.integer :kind, null: false
+      t.references :group, null: false, foreign_key: true
+      t.decimal :balance, precision: 10, scale: 2, default: 0, null: false
+
+      t.timestamps
+    end
+    add_reference :accounts, :owner, foreign_key: { to_table: :users }, null: false
```

Comment:
Another place to remove the account owner (the owner will be the group's owner):

```suggestion
```